includes/class-wc-install.php

```diff
@@ -514,7 +514,7 @@ private static function get_schema() {
 ) $collate;
 CREATE TABLE {$wpdb->prefix}woocommerce_downloadable_product_permissions (
   permission_id BIGINT UNSIGNED NOT NULL auto_increment,
-  download_id varchar(32) NOT NULL,
+  download_id varchar(36) NOT NULL,
```

Comment:
Yea, this is due to the longer UUIDs we'll be using, which are 36 chars long, instead of the 32 char MD5 previously.